pyracf/common/security_admin.py

```diff
@@ -84,6 +93,12 @@ def __overwrite_field_data(self, field_data: dict):
         """Overwrite field data in valid segment traits dictionary"""
         self._valid_segment_traits = field_data
 
+    def __add_additional_secret_traits(self, new_secrets: dict):
```

Comment:
Change `new_secrets` to `additional_secret_traits` for consistency and because `new_secrets` implies that we are overwriting the existing `_secret_traits` dictionary when in reality we are just appending additional secrets to `_secret_traits`.

Reply:
I am not seeing this change in the code.

Reply:
Apologies, that change must have been lost when stashing changes between branches. With later changes this now becomes the following.

```python
    def __add_additional_secret_traits(self, additional_secret_traits: list):
        """Add additional fields to be redacted in logger output"""
        for secret in additional_secret_traits:
            if secret in self.__secret_traits:
                continue
            if ":" not in secret:
                continue
            segment = secret.split(":")[0]
            if segment not in self._valid_segment_traits:
                continue
            if secret not in self._valid_segment_traits[segment]:
                continue
            self.__secret_traits[secret] = self._valid_segment_traits[segment][secret]
```